index.d.ts

```diff
@@ -684,7 +684,7 @@ export declare class Record<ReturnType, PathParams, QueryParams, KeepByInfo, Ext
         /**
          * Path params values
          */
-        path: Dict<PathParams, any>
+        path: Dict<PathParams | 'id' | 'slug', any>
```

Comment:
По умолчанию `slug` может путать, только `id` по умолчанию поддерживаются

```suggestion
        path: Dict<PathParams | 'id', unknown>
```

Reply:
Пришли к решению, что: для типизации path-параметров (при использовании) необходимо предварительно использовать функцию .pathParam для указания разрешенных параметров:
```
public item = Record.new<IOffer>('/api/post/[slug]', {}).pathParam('slug')
````
Добавлена возможность использования функции `pathParam` без уточнения `value` (т.е. только для типизации): теперь 2-й аргумент необязателен.